servers/physics/gjk_epa.cpp

```diff
@@ -722,7 +722,7 @@ struct	GJK
 					append(m_stock,face);
 					return(0);
 				}
-				m_status=m_stock.root?eStatus::OutOfVertices:eStatus::OutOfFaces;
+				m_status=eStatus::OutOfFaces;
```

Comment:
This is thirdparty code mixed in the middle of our own code (hence the different style), so it would be good to mark Godot-made changes as such:
```
// -- GODOT start --
//m_status=m_stock.root?eStatus::OutOfVertices:eStatus::OutOfFaces;
m_status=eStatus::OutOfFaces;
// -- GODOT end --
```